src/index.html

```diff
@@ -26,7 +26,7 @@
     }
   </style>
 </head>
-<body class="ix-dark" style="--bg1: #1e1e1e; background-color: var(--bg1)">
```

Comment:
It was added for initial loading until styles and scripts were loaded. Perhaps we may need to find a less hacky approach to achieve this result or just remove it.

The difference could be shown in this video, `localhost:8080` is `NAS-119098` changes, and `localhost:4200` is `master` branch:

https://user-images.githubusercontent.com/351613/203682727-83679950-86e4-4977-a8b9-36384b68b52e.mp4